crawler.js

```diff
@@ -258,11 +258,12 @@ class Crawler {
     const warcioPackageJson = JSON.parse(await fsp.readFile("/app/node_modules/warcio/package.json"));
     const pywbVersion = child_process.execSync("pywb -V", {encoding: "utf8"}).trim().split(" ")[1];
 
-    const info = {
+    var info = {
       "software": `Browsertrix-Crawler ${packageFileJSON["version"]} (with warcio.js ${warcioPackageJson} pywb ${pywbVersion})`,
       "format": "WARC File Format 1.1"
     };
 
+    info = Object.assign(info, this.params.warcInfo);
```

Comment:
This can just be:

```suggestion
    info = {...this.params.warcInfo, ...info};
```

so we don't allow overriding `software` or `format`